{
  "Faults": [
    {
      "latitude": 51.29186341595618,
      "type": "HV",
      "reference": "KJ1154",
      "etaUpdate": true,
      "affectedCustomerCount": 3,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "9c5e47d0-30d0-4564-bc94-acb8d7b67572",
      "minLongitude": -0.8276,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8233025318659839,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-27T14:59:00.000+0000",
      "estimatedRestoration": "2022-12-28T14:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1154'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1154",
      "maxLongitude": -0.8176,
      "minLatitude": 51.2888,
      "name": "GU51 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU51 2RN"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8276,
              51.2924
            ],
            [
              -0.8176,
              51.2924
            ],
            [
              -0.8176,
              51.2888
            ],
            [
              -0.8276,
              51.2888
            ],
            [
              -0.8276,
              51.2924
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-12-28T08:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2924,
      "etrUpdate": true
    },
    {
      "latitude": 50.79021624164059,
      "type": "LV",
      "reference": "KJ1074",
      "etaUpdate": true,
      "engineerOnSiteTime": "2022-12-27T14:40:00.000+0000",
      "affectedCustomerCount": 22,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "11b6fc1b-3321-4729-9c6f-cb2e3823fee1",
      "minLongitude": -0.6721,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6704141149753707,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-27T13:42:00.000+0000",
      "estimatedRestoration": "2022-12-28T01:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. We are working hard to get the power back on as quickly as we can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1074'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1074",
      "maxLongitude": -0.669,
      "minLatitude": 50.7892,
      "name": "PO21 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO21 1FW",
        "PO21 1HT",
        "PO21 1JA",
        "PO21 1JD",
        "PO21 1JE",
        "PO21 1JF",
        "PO22 8BN",
        "PO22 8BW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6721,
              50.792
            ],
            [
              -0.669,
              50.792
            ],
            [
              -0.669,
              50.7892
            ],
            [
              -0.6721,
              50.7892
            ],
            [
              -0.6721,
              50.792
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-12-27T15:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.792,
      "etrUpdate": true
    },
    {
      "latitude": 51.47682804959149,
      "type": "LV",
      "reference": "KJ1188",
      "etaUpdate": true,
      "engineerOnSiteTime": "2022-12-27T17:00:00.000+0000",
      "affectedCustomerCount": 32,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "8ec38385-3d0b-4b65-94db-f372ca91d42f",
      "minLongitude": -0.6418,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6378776114845853,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-26T20:47:00.000+0000",
      "estimatedRestoration": "2022-12-28T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1188'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1188",
      "maxLongitude": -0.634,
      "minLatitude": 51.4711,
      "name": "SL4 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SL4 4HH",
        "SL4 4JH",
        "SL4 4JJ",
        "SL4 5BB",
        "SL4 5BD",
        "SL4 5BE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.6418,
              51.4794
            ],
            [
              -0.634,
              51.4794
            ],
            [
              -0.634,
              51.4711
            ],
            [
              -0.6418,
              51.4711
            ],
            [
              -0.6418,
              51.4794
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-12-27T17:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.4794,
      "etrUpdate": true
    },
    {
      "latitude": 50.79880236033785,
      "type": "LV",
      "reference": "KJ1128",
      "etaUpdate": true,
      "engineerOnSiteTime": "2022-12-27T15:30:00.000+0000",
      "affectedCustomerCount": 1,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "ea11f3c3-87f0-420c-82c2-b73f464559ac",
      "minLongitude": -0.8272,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8163009240914253,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-27T14:39:00.000+0000",
      "estimatedRestoration": "2022-12-28T13:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1128'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1128",
      "maxLongitude": -0.8042,
      "minLatitude": 50.7917,
      "name": "PO20 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO20 7BX"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8272,
              50.8055
            ],
            [
              -0.8042,
              50.8055
            ],
            [
              -0.8042,
              50.7917
            ],
            [
              -0.8272,
              50.7917
            ],
            [
              -0.8272,
              50.8055
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.8055,
      "etrUpdate": true
    },
    {
      "latitude": 51.09787684377267,
      "type": "LV",
      "reference": "KJ1038",
      "etaUpdate": true,
      "engineerOnSiteTime": "2022-12-27T12:50:00.000+0000",
      "affectedCustomerCount": 4,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "9828b03d-4357-443e-b5ad-9ec44ae4c0b6",
      "minLongitude": -1.0402,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.0327847115932955,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-27T22:29:00.000+0000",
      "estimatedRestoration": "2022-12-28T14:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1038'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1038",
      "maxLongitude": -1.0217,
      "minLatitude": 51.0919,
      "name": "GU34 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU34 5AT",
        "GU34 5AU"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.0402,
              51.1014
            ],
            [
              -1.0217,
              51.1014
            ],
            [
              -1.0217,
              51.0919
            ],
            [
              -1.0402,
              51.0919
            ],
            [
              -1.0402,
              51.1014
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.1014,
      "etrUpdate": true
    },
    {
      "latitude": 50.87815446878352,
      "type": "HV",
      "reference": "KJ1520",
      "etaUpdate": true,
      "affectedCustomerCount": 70,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "bc36a5d3-ea8d-4443-b830-f959d562166c",
      "minLongitude": -0.8585,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6811469573612309,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-28T06:45:00.000+0000",
      "estimatedRestoration": "2022-12-28T15:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1520'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1520",
      "maxLongitude": -0.6419,
      "minLatitude": 50.8583,
      "name": "PO18 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU28 0LZ",
        "PO18 0LD",
        "PO18 0LJ",
        "PO18 0LL",
        "PO18 0LN",
        "PO18 0LP",
        "PO18 0LR",
        "PO18 0LS",
        "PO18 0LT",
        "PO18 0LU",
        "PO18 0LW",
        "PO18 0LX",
        "PO18 0LY",
        "PO18 0LZ",
        "PO18 0NA",
        "PO18 0ND",
        "PO18 0PY",
        "PO18 0QS",
        "PO18 0QZ",
        "PO18 0RS"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8585,
              50.9111
            ],
            [
              -0.6419,
              50.9111
            ],
            [
              -0.6419,
              50.8583
            ],
            [
              -0.8585,
              50.8583
            ],
            [
              -0.8585,
              50.9111
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9111,
      "etrUpdate": true
    },
    {
      "latitude": 51.52494070272709,
      "type": "LV",
      "reference": "KJ1472",
      "etaUpdate": true,
      "engineerOnSiteTime": "2022-12-28T03:50:00.000+0000",
      "affectedCustomerCount": 53,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "6816a7dc-6fda-4d23-b05f-bd870b472713",
      "minLongitude": -0.4217,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.41896205512412976,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-28T03:04:00.000+0000",
      "estimatedRestoration": "2022-12-28T14:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1472'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1472",
      "maxLongitude": -0.4155,
      "minLatitude": 51.5229,
      "name": "UB4 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "UB4 0AA",
        "UB4 0AB",
        "UB4 0AD",
        "UB4 0AE",
        "UB4 0AL",
        "UB4 0AY",
        "UB4 8DG"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.4217,
              51.5269
            ],
            [
              -0.4155,
              51.5269
            ],
            [
              -0.4155,
              51.5229
            ],
            [
              -0.4217,
              51.5229
            ],
            [
              -0.4217,
              51.5269
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5269,
      "etrUpdate": true
    },
    {
      "latitude": 51.51391227676407,
      "type": "LV",
      "reference": "KJ1529",
      "etaUpdate": true,
      "engineerOnSiteTime": "2022-12-28T09:10:00.000+0000",
      "affectedCustomerCount": 31,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "49a7da22-ba9c-40df-8ee8-f9aa47d92fe3",
      "minLongitude": -0.3041,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.30161334035994136,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-28T10:42:00.000+0000",
      "estimatedRestoration": "2022-12-28T14:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1529'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1529",
      "maxLongitude": -0.2991,
      "minLatitude": 51.513,
      "name": "W5 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "W5 2PJ",
        "W5 5JN"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.3041,
              51.5151
            ],
            [
              -0.2991,
              51.5151
            ],
            [
              -0.2991,
              51.513
            ],
            [
              -0.3041,
              51.513
            ],
            [
              -0.3041,
              51.5151
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-12-28T09:15:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.5151,
      "etrUpdate": true
    },
    {
      "latitude": 50.89556277001718,
      "type": "LV",
      "reference": "KJ1614",
      "etaUpdate": true,
      "affectedCustomerCount": 49,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "e7862b1a-9651-4382-9196-52fad7b12e98",
      "minLongitude": -1.3595,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3477688856505996,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-28T09:43:00.000+0000",
      "estimatedRestoration": "2022-12-28T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1614'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1614",
      "maxLongitude": -1.3312,
      "minLatitude": 50.8885,
      "name": "SO19 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO18 5EE",
        "SO19 0NQ",
        "SO19 0NS",
        "SO19 0SX",
        "SO19 9AT",
        "SO19 9BJ",
        "SO19 9LT",
        "SO19 9LX",
        "SO19 9LY",
        "SO19 9LZ",
        "SO19 9PW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.3595,
              50.9157
            ],
            [
              -1.3312,
              50.9157
            ],
            [
              -1.3312,
              50.8885
            ],
            [
              -1.3595,
              50.8885
            ],
            [
              -1.3595,
              50.9157
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-12-28T10:45:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.9157,
      "etrUpdate": true
    },
    {
      "latitude": 51.31284650342303,
      "type": "LV",
      "reference": "KJ1687",
      "etaUpdate": true,
      "affectedCustomerCount": 38,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "1deec3e9-ea85-49e6-bf91-188a493e6337",
      "minLongitude": -0.7341,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.7298901534953144,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-28T10:18:00.000+0000",
      "estimatedRestoration": "2022-12-28T13:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1687'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1687",
      "maxLongitude": -0.7264,
      "minLatitude": 51.3104,
      "name": "GU16 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU16 8DA",
        "GU16 8DF",
        "GU16 8XF",
        "GU16 8XP",
        "GU16 8XR",
        "GU16 8XS",
        "GU16 8XT",
        "GU16 8XU",
        "GU16 9AR"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.7341,
              51.3151
            ],
            [
              -0.7264,
              51.3151
            ],
            [
              -0.7264,
              51.3104
            ],
            [
              -0.7341,
              51.3104
            ],
            [
              -0.7341,
              51.3151
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-12-28T12:00:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.3151,
      "etrUpdate": true
    },
    {
      "latitude": 51.2623630221799,
      "type": "LV",
      "reference": "KJ1731",
      "etaUpdate": true,
      "affectedCustomerCount": 12,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "6c94f0a1-d027-4056-bda5-ffe460f110a6",
      "minLongitude": -0.8318,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8298891324129897,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-28T11:10:00.000+0000",
      "estimatedRestoration": "2022-12-28T17:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1731'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1731",
      "maxLongitude": -0.8274,
      "minLatitude": 51.2609,
      "name": "GU52 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "GU52 8JS",
        "GU52 8JT",
        "GU52 8UY",
        "GU52 8UZ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8318,
              51.2637
            ],
            [
              -0.8274,
              51.2637
            ],
            [
              -0.8274,
              51.2609
            ],
            [
              -0.8318,
              51.2609
            ],
            [
              -0.8318,
              51.2637
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.2637,
      "etrUpdate": true
    },
    {
      "latitude": 51.9642206230194,
      "type": "HV",
      "reference": "KJ1739",
      "etaUpdate": true,
      "affectedCustomerCount": 1107,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "4b9ed4b1-7bb6-4e1d-a60e-cba9998a987b",
      "minLongitude": -1.6003,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.547875477400378,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-28T11:23:00.000+0000",
      "estimatedRestoration": "2022-12-28T15:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1739'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1739",
      "maxLongitude": -1.4872,
      "minLatitude": 51.8639,
      "name": "OX7 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "OX7 5AN",
        "OX7 5FD",
        "OX7 5FE",
        "OX7 5FF",
        "OX7 5FG",
        "OX7 5FL",
        "OX7 5FQ",
        "OX7 5PH",
        "OX7 5PL",
        "OX7 5PP",
        "OX7 5PR",
        "OX7 5PT",
        "OX7 5PU",
        "OX7 5PX",
        "OX7 5PY",
        "OX7 5PZ",
        "OX7 5QA",
        "OX7 5QB",
        "OX7 5QD",
        "OX7 5QE",
        "OX7 5QF",
        "OX7 5QG",
        "OX7 5QT",
        "OX7 5QZ",
        "OX7 5RA",
        "OX7 5RB",
        "OX7 5RD",
        "OX7 5RE",
        "OX7 5RF",
        "OX7 5RG",
        "OX7 5RH",
        "OX7 5RJ",
        "OX7 5RL",
        "OX7 5RN",
        "OX7 5RP",
        "OX7 5RQ",
        "OX7 5RR",
        "OX7 5RS",
        "OX7 5RT",
        "OX7 5RU",
        "OX7 5RW",
        "OX7 5RX",
        "OX7 5RY",
        "OX7 5SA",
        "OX7 5SB",
        "OX7 5SE",
        "OX7 5SN",
        "OX7 5SQ",
        "OX7 5ST",
        "OX7 5SW",
        "OX7 5TF",
        "OX7 5TG",
        "OX7 5TQ",
        "OX7 5XR",
        "OX7 5YJ",
        "OX7 5YL",
        "OX7 5YN",
        "OX7 5YP",
        "OX7 5YR",
        "OX7 5YS",
        "OX7 5YT",
        "OX7 5YU",
        "OX7 5YW",
        "OX7 5YX",
        "OX7 5YY",
        "OX7 5YZ",
        "OX7 6AW"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.6003,
              51.994
            ],
            [
              -1.4872,
              51.994
            ],
            [
              -1.4872,
              51.8639
            ],
            [
              -1.6003,
              51.8639
            ],
            [
              -1.6003,
              51.994
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 51.994,
      "etrUpdate": true
    },
    {
      "latitude": 50.980074378384636,
      "type": "LV",
      "reference": "KJ1585",
      "etaUpdate": true,
      "engineerOnSiteTime": "2022-12-28T10:40:00.000+0000",
      "affectedCustomerCount": 3,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "1cb54a86-9bc1-47be-8a50-75cb132bcd7f",
      "minLongitude": -1.6097,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.6046305042127782,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-28T09:18:00.000+0000",
      "estimatedRestoration": "2022-12-28T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1585'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1585",
      "maxLongitude": -1.5994,
      "minLatitude": 50.9773,
      "name": "SO51 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "SO51 6EF"
      ],
      "location": {
        "coordinates": [
          [
            [
              -1.6097,
              50.985
            ],
            [
              -1.5994,
              50.985
            ],
            [
              -1.5994,
              50.9773
            ],
            [
              -1.6097,
              50.9773
            ],
            [
              -1.6097,
              50.985
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.985,
      "etrUpdate": true
    },
    {
      "latitude": 50.83561108794717,
      "type": "LV",
      "reference": "KH9721",
      "etaUpdate": true,
      "engineerOnSiteTime": "2022-12-24T13:20:00.000+0000",
      "affectedCustomerCount": 18,
      "networkId": "com.sse.ssepd.sepd",
      "UUID": "26e80ce0-e85d-47ec-af5c-0be6e522cbea",
      "minLongitude": -0.8119,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.8101867114961472,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-24T11:42:00.000+0000",
      "estimatedRestoration": "2022-12-28T17:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KH9721'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KH9721",
      "maxLongitude": -0.7991,
      "minLatitude": 50.8319,
      "name": "PO19 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "PO19 3FY",
        "PO19 3JJ",
        "PO19 3JL"
      ],
      "location": {
        "coordinates": [
          [
            [
              -0.8119,
              50.8387
            ],
            [
              -0.7991,
              50.8387
            ],
            [
              -0.7991,
              50.8319
            ],
            [
              -0.8119,
              50.8319
            ],
            [
              -0.8119,
              50.8387
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 50.8387,
      "etrUpdate": true
    },
    {
      "latitude": 57.46078941147073,
      "type": "LV",
      "reference": "KJ1570",
      "etaUpdate": true,
      "affectedCustomerCount": 28,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "5aa29629-54f8-459c-8640-95322fb5f761",
      "minLongitude": -4.2272,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -4.222386050041144,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-28T08:49:00.000+0000",
      "estimatedRestoration": "2022-12-28T14:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1570'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1570",
      "maxLongitude": -4.2189,
      "minLatitude": 57.459,
      "name": "IV2 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "IV2 4NP",
        "IV2 4NR",
        "IV2 4NS",
        "IV2 4NT",
        "IV2 4NU",
        "IV2 4NW",
        "IV2 4NX",
        "IV2 4NY"
      ],
      "location": {
        "coordinates": [
          [
            [
              -4.2272,
              57.4624
            ],
            [
              -4.2189,
              57.4624
            ],
            [
              -4.2189,
              57.459
            ],
            [
              -4.2272,
              57.459
            ],
            [
              -4.2272,
              57.4624
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-12-28T09:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.4624,
      "etrUpdate": true
    },
    {
      "latitude": 56.591856044735465,
      "type": "LV",
      "reference": "KJ1611",
      "etaUpdate": true,
      "affectedCustomerCount": 2,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "d87132f1-577f-4054-bbbf-ce941d3f6120",
      "minLongitude": -2.7418,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.725027324097396,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-28T09:39:00.000+0000",
      "estimatedRestoration": "2022-12-28T16:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1611'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1611",
      "maxLongitude": -2.7135,
      "minLatitude": 56.5873,
      "name": "DD11 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "DD11 2RE"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.7418,
              56.6007
            ],
            [
              -2.7135,
              56.6007
            ],
            [
              -2.7135,
              56.5873
            ],
            [
              -2.7418,
              56.5873
            ],
            [
              -2.7418,
              56.6007
            ]
          ]
        ],
        "type": "Polygon"
      },
      "estimatedArrivalOnSiteTime": "2022-12-28T10:30:00.000+0000",
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 56.6007,
      "etrUpdate": true
    },
    {
      "latitude": 57.149914803458984,
      "type": "LV",
      "reference": "KJ1642",
      "etaUpdate": true,
      "engineerOnSiteTime": "2022-12-28T11:00:00.000+0000",
      "affectedCustomerCount": 3,
      "networkId": "com.sse.ssepd.shepd",
      "UUID": "bb6f1da5-0ed6-41c1-b2ad-597b3556a486",
      "minLongitude": -2.2113,
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -2.2022965505135894,
      "resolved": false,
      "py/object": "Fault",
      "srtUpdate": true,
      "loggedAt": "2022-12-28T10:01:00.000+0000",
      "estimatedRestoration": "2022-12-28T13:30:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. Our engineers are on site working hard to get the power back on as quickly as they can. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'KJ1642'",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/KJ1642",
      "maxLongitude": -2.1938,
      "minLatitude": 57.1469,
      "name": "AB15 Area",
      "otherUpdate": true,
      "affectedAreas": [
        "AB15 8PP",
        "AB15 8PQ"
      ],
      "location": {
        "coordinates": [
          [
            [
              -2.2113,
              57.1536
            ],
            [
              -2.1938,
              57.1536
            ],
            [
              -2.1938,
              57.1469
            ],
            [
              -2.2113,
              57.1469
            ],
            [
              -2.2113,
              57.1536
            ]
          ]
        ],
        "type": "Polygon"
      },
      "region": "com.opengrid.geo.world.europe.gb",
      "maxLatitude": 57.1536,
      "etrUpdate": true
    }
  ]
}
